src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationInvalidSwitchIndent.java

```diff
@@ -101,6 +101,10 @@ private void method1() { //indent:4 exp:4
 	    		break; //indent:16 exp:20 warn
 	    	else //indent:12 exp:16 warn
 	    		break; //indent:16 exp:20 warn
+            case 4: //indent:12 exp:12
+                s++; //indent:16 exp:16
+    /* hello */ case 5: //indent:4 exp:12 warn
```

Comment:
> When are these 2 values not equal? It seems to me when they are not equal there is an issue of some kind.

Here `getLineStart()` gives `4` and `expandedTabscolumnNumber()` gives `16`.
Is it an Issue of some kind.

Reply:
@romani This does seem like an issue since 2 different methods which sound like they are doing the same thing return different values, especially if they should be used interchangablly. I don't remember how we said we would handle cases like this. I thought the comment indentation would be what we validate for the line but can't remember for sure.

Reply:
Indentation Check is waiting for long reimplementation. we have big pack of issues on it, expected behavior is not clear.